src/gitHubIssueFetcher.ts

```diff
@@ -62,29 +75,61 @@ query findByLabel($queryString:String!, $pageSize:Int, $after:String) {
             endCursor
         }
     }
+    rateLimit {
+        limit
+        cost
+        remaining
+        resetAt
+    }
 }`;
 
 export class GitHubIssueFetcher {
-    private readonly logger: mariner.Logger;
+    private readonly logger: Logger;
 
-    public constructor(logger: mariner.Logger) {
+    public constructor(logger: Logger) {
         this.logger = logger;
     }
 
     public async fetchMatchingIssues(
         token: string,
         label: string,
         repositoryIdentifiers: string[]
-    ): Promise<IssueCountAndIssues> {
-        const listOfRepos = this.createListOfRepos(repositoryIdentifiers);
-        const variables: Variables = {
-            // NOTE: See https://docs.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests
-            queryString: `label:\"${label}\" state:open ${listOfRepos}`,
-            pageSize: 10,
-        };
-        const issueCountAndIssues = await this.fetchAllPages(token, query, variables);
+    ): Promise<GitHubIssue[]> {
+
+        const pageSize = 100;
+        const numberOfReposPerCall = 1000;
+        const chunks = this.splitArray(repositoryIdentifiers, numberOfReposPerCall);
```

Comment:
@IngridGdesigns : I wanted to call out that you made a great choice passing the number of repos as a parameter, as opposed to having the constant inside the split function.